unity/ComputerVision-UnityClient/Assets/Scripts/AppManager.cs

```diff
@@ -0,0 +1,370 @@
+using UnityEngine;
+using System.Collections;
+using UnityEngine.UI;
+using UnityEngine.Networking;
+using System.Threading.Tasks;
+using System;
+
+namespace MobiledgeXComputerVision
+{
+    public class AppManager : MonoBehaviour
+    {
+        public RawImage rawImage;
+        public Texture rectTexture;
+        public Texture nodeTexture;
+        public static bool showGUI;
+        int[][] faceDetectionRects;
+        int[] faceRecognitionRect;
+        static string faceRecognitionSubject;
+        static float faceRecognitionConfidenceLevel;
+        static @Object[] objectsDetected;
+        float imgScalingFactor;
+        public static int level = 0;
+        public NetworkManager networkManager;
+        bool serviceAlreadyStarted
+        {
+            get
+            {
+                switch (serviceMode)
+                {
+
+                    case ServiceMode.FaceRecognition:
+                        return faceRecognitionRect == null ? false : true;
+                    case ServiceMode.ObjectDetection:
+                        return objectsDetected == null ? false : true;
+                    default:
```

Comment:
default before the last case?

Reply:
So I think its better performance to have default in the last case  the msdn  references are doing so.
https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/switch

Reply:
So it should be case, case, case, default (right?)

Reply:
Yes, I guess you have a point, I will default to false after cases